src/components/search-box.jsx

```diff
@@ -0,0 +1,55 @@
+import React, {PropTypes} from "react"
+
+
+const SearchBox = React.createClass({
+  propTypes: {
+    labelFilter: PropTypes.string,
+    nameFilter: PropTypes.string,
+    onLabelChange: PropTypes.func,
+    onNameChange: PropTypes.func,
+  },
+  onLabelInputChange(event) {
+    this.props.onLabelChange(event.target.value)
+  },
+  onNameInputChange(event) {
+    this.props.onNameChange(event.target.value)
```

Comment:
I have limited experience with React, but I thought one of its points was to remove that kind of event forwarding 🤔